packages/react-app/components/cards/MediaCard.jsx

```diff
@@ -70,27 +72,55 @@ function MediaCard({
   const toast = useToast();
 
   const handleRequestPrivateProfileUnlock = async () => {
+    const stakeAmount = prompt("Enter stake amount in MATIC");
+    if (!stakeAmount) {
+      return alert("Please enter a valid stake amount");
```

Comment:
Perform validation using the NumberInput component: https://chakra-ui.com/docs/form/number-input